src/bundle/Resources/public/js/OnlineEditor/buttons/ez-btn-styleslist.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import AlloyEditor from 'alloyeditor';
+import EzButtonDropdown from './ez-btn-dropdown';
+
+export default class EzButtonStylesList extends AlloyEditor.ButtonStylesList {
+
+    render() {
+        var e;
+        return this.props.showRemoveStylesItem && (e = React.createElement(AlloyEditor.ButtonStylesListItemRemove, {
```

Comment:
Can you use the JSX syntax?

Reply:
@dew326

> To fix this in 2.5 it should be opened also to admin-ui bundle
Sure, I'm aware, after its fine here I'm gonna post 2nd PR to admin-ui-bundle

> Can you use the JSX syntax?
Yes, however, this `render` method is taken directly out of AlloyEditor, I wanted to keep it the same, should I reformat it?

Reply:
From where did you copy this method?

Reply:
```
createReactClass({
                displayName: "ButtonStylesList"
```
Its basically this code fragment of alloy-editor-no-react-min.js - I know its minified, but its just render method of `ButtonStylesList` component.

Reply:
If you want to copy something please always take the source:
https://github.com/liferay/alloy-editor/blob/1.x/src/ui/react/src/components/buttons/button-styles-list.jsx